### ISSUE TITLE
Error for generating WebApi project.

### ISSUE THREAD

**Issue by vzhuqin** [dotnet/machinelearning-modelbuilder#1664]
**System Information (please complete the following information):**
- Windows OS: Windows-10-Enterprise-21H1
- ML.Net Model Builder (Preview): 16.7.0.2137701 (Main Branch)
- Microsoft Visual Studio Enterprise 2019: 16.10.4
- .Net: 5.0.302
![image](https://user-images.githubusercontent.com/81727020/127825309-ae038a0b-bd52-4f8e-beb8-0b3d132fe1f1.png)

**Describe the bug**
- On which step of the process did you run into an issue: Generating WebApi project
- Clear description of the problem: Is this a new update? the target framework of the generated WebApi project decided to use net6?

**To Reproduce**
1. Select Create a new project from the Visual Studio 2019 start window;
2. Choose the C# Console App (.NET Core) project template with .Net 5.0;
3. Add model builder by right click on the project;
4. Select Text classification to train complete;
5. Navigate to Consume page, click "Add to solution" after Web API;
6. See error during generating: The current .NET SDK does not support targeting .NET 6.0;
7. Open the .csproj file to check the target framework, find that the version is net6.0

**Expected behavior**
The version of the target framework is net5.0

**Screenshots**
![image](https://user-images.githubusercontent.com/81727020/127823617-8bfa471b-62b8-400e-8c80-8c6927f5ba04.png)

**Additional context**
This issue not occurs on the generated ConsoleApp project.

**Comment by LittleLittleCloud** [comment]
@briacht @zewditu 
Should we target net5 for all projects if I recall correctly?

**Comment by zewditu** [comment]
Yes, we were targeted to net5, however recently  our web API is targeted  to NET 6 in this PR https://github.com/dotnet/machinelearning-tools/pull/1136 to bring  NET6 OpenAPIsupport  https://github.com/dotnet/machinelearning-modelbuilder/issues/1605

**Comment by zewditu** [comment]
The thing I am not sure is whose responsibility is to upgrade the SDK to support .NET 6? the user? or is there any thing we can do ? @LittleLittleCloud  thought?

**Comment by beccamc** [comment]
@JakeRadMSFT To comment on correct approach

**Comment by vzhuqin** [comment]
Not repro this issue on:
- Microsoft Visual Studio Enterprise 2019: 16.11.3
- ML.Net Model Builder: 16.7.4.2146502 (Main Branch)

![image.png](https://images.zenhubusercontent.com/60653e492390a08ea616cb10/9833452f-101f-4d75-9b2d-53d56a7bc0fe)
![image.png](https://images.zenhubusercontent.com/60653e492390a08ea616cb10/3953aafa-01e7-4885-87f3-067028f80641)
